### DISCUSSION TITLE
Next planning session suggestion: March 6th

### DISCUSSION THREAD

**Discussion by bmann** [DWebYVR/notes#11] [Planning & Suggestions]
I’m suggesting ~~Feb 28th~~ March 6th as the next planning session, this time at Revolver https://lu.ma/em94x8fo

I propose we leave comments in this thread for the agenda.

**Comment by bmann** [comment]
@jackyzha0 does this time work for you / do you have interest in joining us in person?

**Reply by jackyzha0** [reply]
Yup! Can join in person
